web-wallet/src/lib/stores/__tests__/networkStore.spec.js

```diff
@@ -69,4 +69,18 @@ describe("Network store", async () => {
 
     await expect(store.getCurrentBlockHeight()).resolves.toBe(blockHeight);
   });
+
+  it("should expose a service method to retrieve a `AddresszSyncer` for the network", async () => {
```

Comment:
A typo – `AddressSyncer`.

Reply:
Oh, thanks. And appreciated the nitpick, as always!